carberretta/bot/cogs/experience.py

```diff
@@ -0,0 +1,80 @@
+import typing
```

Comment:
```suggestion
import typing as t
```
Don't forget to change other instances.